tfhe-zk-pok/src/proofs/pke_v2.rs

```diff
@@ -2128,7 +2142,7 @@ mod tests {
                 PublicParams::deserialize_with_mode(data.as_slice(), compress, Validate::No)
             };
 
-        let original_public_param = crs_gen_ghl::<Curve>(d, k, B, q, t, rng);
+        let original_public_param = crs_gen_ghl::<Curve>(d, 2 * k, B, q, t, rng);
```

Comment:
Maybe we should use `crs_gen_cs` since that's the one used in the impl, or simply `crs_gen`

Reply:
yep you are right here

Reply:
changed the call to have crs_gen here instead, changed the 2 * k to something giving a random value bigger than k and <= to d